### PR TITLE
refactor: move llm bots to separate package

### PR DESCRIPTION
**Pull request by md0x**
**Motivation**

Move LLM bots to a separate package

### CODE REVIEW

File: packages/llm-bot/test/constants.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { formatBytes32String, parseUnits } from "./utils";
```
Comment (mrice32) [inline packages/llm-bot/test/constants.ts:1]: Are some/all of these test utilities files copied? If so, could you comment that in the PR so we know which to review as fresh code?

File: packages/llm-bot/src/utils/logger.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { TenderlySimulationResult } from "@uma/common";
+
+const optimisticOracleV2UIBaseUrl = "https://oracle.uma.xyz";
+const testnetOptimisticOracleV2UIBaseUrl = "https://testnet.oracle.uma.xyz";
+
+// monitor-v2 package is only using Optimistic Oracle V3, so currently there is no need to generalize this.
```
Comment (mrice32) [inline packages/llm-bot/src/utils/logger.ts:6]: nit: outdated comment

File: packages/llm-bot/src/utils/logger.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    return "No Tenderly simulation available";
+  } else if (simulationResult.status) {
+    return `<${simulationResult.resultUrl.url}|Tenderly simulation successful${
+      !simulationResult.resultUrl.public ? " (private)" : ""
+    }>`;
+  } else {
+    return `<${simulationResult.resultUrl.url}|Tenderly simulation reverted${
+      !simulationResult.resultUrl.public ? " (private)" : ""
+    }>`;
+  }
```
Comment (mrice32) [inline packages/llm-bot/src/utils/logger.ts:28]: Was a little surprised to see this code here? Is this used (or placed to be used) in the LLC bit codebase? If so, no problem, just wanted to call it out!

**Review by mrice32 — APPROVED** [review]
LGTM! Only nits and minor comments

**Review by daywiss — APPROVED** [review]
im assuming this is just moving stuff, but i dont see why the changes from old location arent showing

**Pull request merged**